README.md

```diff
@@ -1,6 +1,96 @@
-# odin-recipes
-Microverse Pre-Enrollment Bootcamp assignment on HTML-CSS :=> Odin Recipes
+<a name="readme-top"></a>
 
-```text
-Coded By: Md Mahabub A. (~/gh/mahabubx7)
-```
+<div align="center">
+  <br/>
+  <h3><b>Odin Recipes - Code review practice project </b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Odin Recipes - Code review practice project] <a name="about-project"></a>
+
+
+**[Odin Recipes - Code review practice project]** is my first project to do the code review practice.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
```

Comment:
This is not mandatory but I would suggest giving the actual content in your `README.md` markdown file.